content/blog/2020/trying-something-new.md

```diff
@@ -2,38 +2,36 @@ Title: Trying something different this year
+## Submissions this year
 
-As you know, PyCon India 2020 is going online this year. With the current situation, the team wanted to take the unique opportunity to make our conference bigger, more inclusive, and much more accessible to our attendees all over the world. We received an oustanding response to the event, with a total of 243 talks submissions under 12 primary domains.
+As you know, [PyCon India 2020](https://in.pycon.org) is going online and completely virtual this year. Hence, the team wanted to take the unique opportunity to make the conference bigger, more inclusive, and much more accessible to our attendees the world over. And, we were successful. We were so glad to see the community's response to our efforts, with outstanding submissions to our [call for proposals](https://in.pycon.org/cfp/2020/proposals/). We finally ended up with a total of 243 talks submitted under 12 primary domains with talks pouring in from all over the world.
 
-### Difficult decision to choose amongst so many remarkable proposals
+## With great proposals comes difficult decisions
 
-We got many excellent talks this year, but we had limited slots available for the conference. So, we had to reject a lot of great talks. The review team had to consider various factors while selecting the talks, including a proper mix of beginner and advanced topics, providing an opportunity for first-time speakers, diversity, etc.
+The **only downside** to getting excellent talks each year at PyCon India is that we end up rejecting many talks. This is because, we are almost always under the burden of limited slots available for the conference. The review team is under great stress of having to consider various factors while selecting the talks, including a proper mix of beginner and advanced topics, providing an opportunity for first-time speakers, diversity, all the while maintaining talk quality, etc.
 
-### Adding an additional track to the conference
+## Our first order of business, adding another track!
 
-PyCon India 2020, over the last few years, has been a three-track event. Seeing the quality of the talks, the team decided to add an additional track for the event this year.
+Over the last few years, PyCon India 2020 has been a three-track event. Seeing the quality of the talks this year, we felt inclined to make this year's PyCon India's lineup our biggest ever with an additional track for the event this year! 
 
-So, PyCon India 2020 will be a four-track event. This would mean the event would have even more amazing speakers presenting their exceptional talks at the conference. The talks would range from beginner to advanced and cover a wide range of computer science topics.
+Hence, PyCon India 2020 will now be a four-track event effectively increasing talks selected for the conference by 33%. This would mean more people getting the opportunity to present, and more talks for us to watch right at our home. Right? Right? We didn't stop there, presenting...!
```

Comment:
5 track? 
Including .extend we have 5. Saying 4 and announcing 5th sounds confusing as @inovizz said.

Reply:
Read the next paragraph.

Reply:
We are going for an incremental change in our tracks. Hence we first introduce the new 4th track to warm audiences up of how it was before and then introduce something entirely new.

Reply:
Yes, I agree with @vipulgupta2048 here. It's incremental. Contextual understanding